backend/settings.py

```diff
@@ -217,9 +217,36 @@
 TINYMCE_DEFAULT_CONFIG = {
         'menubar': True,
         "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
-        "toolbar": "formatselect | bold italic backcolor | "
+        "toolbar": "bold italic backcolor | "
             "bullist numlist outdent indent | removeformat | code | help",
         "default_link_target": "_blank",
+        "invalid_elements": "span, h1, h2, h3, h4, h5, h6",
+        "invalid_styles": "color, font-family, background-color",
+
+        "style_formats": [
+            { "title": 'Inline', "items": [
+            { "title": 'Bold', "format": 'bold' },
+            { "title": 'Italic', "format": 'italic' },
+            { "title": 'Underline', "format": 'underline' },
+            { "title": 'Strikethrough', "format": 'strikethrough' },
+            { "title": 'Superscript', "format": 'superscript' },
+            { "title": 'Subscript', "format": 'subscript' },
+            { "title": 'Code', "format": 'code' }
+            ]},
+            { "title": 'Blocks',  "items": [
+            { "title": 'Paragraph', "format": 'p' },
+            { "title": 'Blockquote', "format": 'blockquote' },
+            { "title": 'Div', "format": 'div' },
+            { "title": 'Pre', "format": 'pre' }
+            ]},
+            { "title": 'Align',  "items": [
+            { "title": 'Left', "format": 'alignleft' },
+            { "title": 'Center', "format": 'aligncenter' },
+            { "title": 'Right', "format": 'alignright' },
+            { "title": 'Justify', "format": 'alignjustify' }
+            ]}
+            ],
+            "block_formats": "Paragraph=p",
```

Comment:
I'll test this today. I'd like to see this spacing normalized so it's easier to read. I'll take a stab at it.

Reply:
So I'd like to see some formatting changes here. First, all this should be unindented once (it has two now). Then, it'd be nice to space the objects and arrays like below for better readability.
```
    'menubar': True,
    "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
    "toolbar": "bold italic backcolor | "
        "bullist numlist outdent indent | removeformat | code | help",
    "default_link_target": "_blank",
    "invalid_elements": "span, h1, h2, h3, h4, h5, h6",
    "invalid_styles": "color, font-family, background-color",
    "style_formats": [
        {
            "title": 'Inline',
            "items": [
                {"title": 'Bold', "format": 'bold'},
                {"title": 'Italic', "format": 'italic'},
                {"title": 'Underline', "format": 'underline'},
                {"title": 'Strikethrough', "format": 'strikethrough'},
                {"title": 'Superscript', "format": 'superscript'},
                {"title": 'Subscript', "format": 'subscript'},
                {"title": 'Code', "format": 'code'}
            ]
        },
        {
            "title": 'Blocks', 
            "items": [
                {"title": 'Paragraph', "format": 'p'},
                {"title": 'Blockquote', "format": 'blockquote'},
                {"title": 'Div', "format": 'div'},
                {"title": 'Pre', "format": 'pre'}
            ]
        },
        {
            "title": 'Align',
            "items": [
                {"title": 'Left', "format": 'alignleft'},
                {"title": 'Center', "format": 'aligncenter'},
                {"title": 'Right', "format": 'alignright'},
                {"title": 'Justify', "format": 'alignjustify'}
            ]
        }
    ],
    "block_formats": "Paragraph=p",
```
I'd also like to see consistent use of single or double quotation marks, but we're not consistent elsewhere in `settings.py`, so it's not the biggest deal.

Reply:
I was hoping there'd be a way to not have to specify all this and just tell it to omit the heading menu, but I haven't found that yet.